src/panels/lovelace/editor/view-editor/hui-edit-view.ts

```diff
@@ -391,7 +391,7 @@ export class HuiEditView extends LitElement {
         .preview-badges {
           display: flex;
           justify-content: center;
-          margin: 8px 16px;
+          margin: 12px 16px;
```

Comment:
Is this the same as on the Lovelace view?

Reply:
It has the 4 px padding around the view. Plus the 8px 16px margin